server/models/tag.js

```diff
@@ -6,6 +6,16 @@ const tagSchema = mongoose.Schema({
 
 const TagModel = mongoose.model('Tag', tagSchema);
 
+saveTag = function(tagData) {
```

Comment:
For consistency we might wanna stick with `function name(input) {}` rather than `name = function(input){}`